src/App.js

```diff
@@ -19,13 +20,32 @@ export default function App() {
                 item.name.toLowerCase().includes(userInput.toLowerCase())
               )
               .map(({ name, image }) => (
-                <AttractionCard key={name} name={name} image={image} />
+                <AttractionCard
+                  key={name}
+                  name={name}
+                  image={image}
+                  likePlace={likePlace}
```

Comment:
`likePlace` is a function? That was not clear at first glance. Maybe consider something like this? 
Remember, with our bookmark function we often had it like this as well `onBookmark={handleBookmark}`, I think.
```suggestion
                  onLike={handleLikePlace}
```

Reply:
Very good point 👍🏻